tck/running_the_tck.asciidoc

```diff
@@ -14,38 +15,91 @@
 
 = Running the Microprofile LRA TCK
 
-The TCK provides a JAX-RS participant resource together with a resource for invoking the test sutie.
+The TCK uses `JUnit 4` and `Arquillian`.
+
+== Configuration
+
+The LRA TCK suite can be parametrized by following properties, handled in the suite with MicroProfile Config
+
+`lra.tck.base.url`::
+  The URL where the TCK suite deployment is exposed at. The TCK suite will construct path based on this URL.
+  The default base URL is `http://localhost:8180`.
+`lra.tck.timeout.factor`::
+  Timeout factor of default value `1.0`. When set bigger than `1.0` then timeout value will be bigger and waiting time are longer.
+  When set-up lower then the timeouts will be shorter.
+`lra.tck.coordinator.waiting`::
+  Time that the tests waits for the LRA recovery starts-up.
+`lra.http.host`, `lra.http.port`::
```

Comment:
Could we use an authority strings for endpoints instead (https://tools.ietf.org/html/rfc3986#section-3.2) or maybe just say it is a URI which gives us maximum flexibility.

Reply:
this parameters will be removed completely as discussed at the meeting the coordinator should not be expected by TCKs